src/DbConnection.cpp

```diff
@@ -1,3 +1,4 @@
+#include <fstream>
 #include "pch.h"
```

Comment:
```suggestion
#include "pch.h"
#include <fstream>
```

Would that work too?